pkg/constants/constants.go

```diff
@@ -27,6 +27,7 @@ import (
 const (
 	GrubConf                     = "/etc/cos/grub.cfg"
 	GrubOEMEnv                   = "grub_oem_env"
+	GrubEnv                      = "grubenv" // grubenv is a file used by grub to store environment variables. Goes into OEM so its runtime modifiable
```

Comment:
What is the difference with `grub_oem_env` above? Maybe document the above file too, to make it clear what each one is used for?

Reply:
grub_oem_env seems to be used only for the install part, and it goes in the state folder, so it should be RO, while this one goes into OEM so its RW during runtime.

I have no idea why we would have 2, we may need to revisit this at some point to merge them? The grub_oem_env is not documented at all so it seems like its use its internal to tweak the grub behaviour ourselves or something? Unclear to me